orchestrator/api/api_v1/endpoints/processes.py

```diff
@@ -76,14 +76,18 @@
 
 
 @router.delete("/{pid}", response_model=None, status_code=HTTPStatus.NO_CONTENT)
-def delete(pid: UUID) -> None:
-    count = ProcessTable.query.filter_by(pid=pid).delete()
-    db.session.commit()
-    if count > 0:
-        return None
-    else:
+def delete(request: Request, pid: UUID) -> None:
+    process = ProcessTable.query.filter_by(pid=pid).one_or_none()
+    if not process:
         raise_status(HTTPStatus.NOT_FOUND)
 
+    broadcast_func = api_broadcast_process_data(request)
+    websocket_data = {"process": {"id": process.pid, "status": ProcessStatus.ABORTED}}
```

Comment:
Why aborted? :thinking:

Reply:
In the frontend we remove processes that have `ABORTED` or `COMPLETED`. there also isn't a `DELETED` because the process doesn't exist anymore :3